include/zephyr/bluetooth/hci_types.h

```diff
@@ -558,6 +559,43 @@ struct bt_hci_cp_io_capability_neg_reply {
 	uint8_t   reason;
 } __packed;
 
+#define BT_HCI_OP_SWITCH_ROLE                   BT_OP(BT_OGF_LINK_POLICY, 0x000b)
+struct bt_hci_cp_switch_role {
+	bt_addr_t bdaddr;
```

Comment:
Any idea why this is taking bdaddr and not a handle?

Reply:
The HCI cmd is used for connection, honestly, I don't know why it doesn't use connection handle.